### ISSUE TITLE
Remove "in 2016" from thank-you video pages to make them evergreen

### ISSUE THREAD

**Issue by WillatMozFdn** [MozillaFoundation/Advocacy#1016]
Detailed description of work:
We need to remove the string "in 2016" from the headline of three pages listed below, so that as a fundraiser, I can re-use these videos in general fundraising thank-you messages this year as well.

https://donate.mozilla.org/en-US/jan-thank-you-superhero
https://donate.mozilla.org/en-US/jan-thank-you-sincere
https://donate.mozilla.org/en-US/jan-thank-you-cute

These pages are also localized in FR, ES, DE and IT -- I'd like to get "in 2016" removed in those strings as well, and am working to figure out the right translations done for that.  Will update this ticket as soon as those are available.

@ScottDowne 
@hannahkane 

*Start date: August 1, 2017* 
*Due date: August 15, 2017* 
Note: The dates need to be exactly in the format indicated above for proper integration with the Gantt chart.

CC: @TheoChevalier

**Comment by ScottDowne** [comment]
There is no good way to target just FR, ES, DE and IT, so this change gets done in all languages, I believe.

**Comment by TheoChevalier** [comment]
We mostly used FR, ES, DE and IT, but yes in reality it’s localized in all languages. Let us know when you have the updated copy and we can get this localized :) Note that we can also keep the year in the new strings and use a variable, if we want to keep the same copy over years (but I believe we want to get a new thank you message every year).

**Comment by ScottDowne** [comment]
@TheoChevalier when can we send this to production?

**Comment by TheoChevalier** [comment]
@ScottDowne we can, it’s not even linked anywhere in the website :)

**Comment by ScottDowne** [comment]
Perfect, shipping!

**Comment by ScottDowne** [comment]
Done! Closing.

**Comment by WillatMozFdn** [comment]
Hey @ScottDowne -- the links to these videos are now (as of this past weekend afaict) all broken?   Received a donor complaint that we're tricking them by sending them back to the donate page ... which is in fact what happens when no content is found that these urls, it resolves back to the root of the subdomain and shows the donate page again.  That's definitely sub-optimal.... (cc @TheoChevalier )

**Comment by TheoChevalier** [comment]
Ugh, looks like we jumped the gun here… https://github.com/mozilla/donate.mozilla.org/pull/1996#issuecomment-392759351
Looks like Scott and I didn’t realize the pages were still in use, and now the translations are gone… But can still be manually recovered from GitHub history.

As a stop gap solution, maybe we can redirect them to https://donate.mozilla.org/en-US/share ?

**Comment by ScottDowne** [comment]
I see, yeah those pretty much got removed because they had "jan thank you" and I forgot we wanted them to be evergreen. With a title like that, they don't sound very evergreen.

We should probably come up with an ever green url so it's clear these are in use. @WillatMozFdn if you come up with a few evergreen names, I can set them up, and redirect the old links to the new ones.

@TheoChevalier and is it best if you re add the strings?

**Comment by WillatMozFdn** [comment]
Well, the strings are not really important ... but having the videos available at those URLs certainly is.  Every time a Stripe donation is made, we are sending those video links out, so this is a live/active/ongoing problem....

**Comment by ScottDowne** [comment]
@WillatMozFdn yeah, we'll probably land these bits as they are done, so as to not have one block the other.

**Comment by WillatMozFdn** [comment]
a more evergreen folder structure would be donate.mozilla.org/thankyou/superhero, donate.mozilla.org/thankyou/sincere , and donate.mozilla.org/thankyou/funny -- if you still need that info.

**Comment by ScottDowne** [comment]
@WillatMozFdn @TheoChevalier this puts the links back the way they were. https://github.com/mozilla/donate.mozilla.org/pull/2004

**Comment by ScottDowne** [comment]
@WillatMozFdn going with http://donate.mozilla.org/en-US/thank-you-superhero/ etc if it's all the same to you.

**Comment by WillatMozFdn** [comment]
Totally fine, Scott -- thanks!!

**Comment by ScottDowne** [comment]
Should be live in a bit, I just deployed.

**Comment by ScottDowne** [comment]
Now live: https://donate.mozilla.org/en-US/thank-you-superhero/
